src/sprites/clone.js

```diff
@@ -31,6 +33,18 @@ function Clone(sprite, runtime) {
      * @type {?Number}
      */
     this.drawableID = null;
+    /**
+     * Dictionary of variables and their values for this clone.
+     * Key is the variable name.
+     * @type {Object.<string,*>}
+     */
+    this.variables = {};
+    /**
+     * Dictionary of lists and their contents for this clone.
+     * Key is the list name.
+     * @type {Object.<string,*>}
+     */
+    this.lists = {};
```

Comment:
Maybe these and `lookupOrCreateVariable`, `lookupOrCreateList` should actually be provided on the superclass, `Target`?